CHANGELOG.md

```diff
@@ -2,8 +2,13 @@
 <!-- Add CHANGELOG entry to this section for any PR awaiting the next release -->
 <!-- Please also include if this is a Bug Fix, Enhancement, or Feature -->
 
+BREAKING CHANGES:
```

Comment:
I still don't believe this rises to the level of a breaking change, but do think it deserves a special note. I'll think on it some more.